puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/ansible.groovy

```diff
@@ -31,8 +31,14 @@ def runPlaybook(args) {
 }
 
 def writeExtraVars(args) {
-    def timestamp = new Date().getTime()
-    def extra_vars_file = 'extra_vars-' + timestamp.toString() + '.yaml'
+    def suffix
+    try {
+        def timestamp = new Date().getTime()
+        suffix = timestamp.toString()
+    } catch(Exception ex) {
+        suffix = 'no_timestamp'
```

Comment:
Should we fall back to a shell implementation?

Reply:
mhhh. we could.

Reply:
Alternatives are finding a job ID but I don't know if you have access to env vars.